joblib/test/test_parallel.py

```diff
@@ -940,3 +940,42 @@ def __reduce__(self):
     with warns(DeprecationWarning):
         with raises(ValueError):
             delayed(UnpicklableCallable(), check_pickle=True)
+
+
+def test_backend_reset():
+    """Test that a backend is correctly reset to its initial state."""
+
+    # let's tolerate 30% error between compuration times to avoid random
+    # failures because of variable available ressources on CIs.
+    error = 0.3
+    n_jobs = 4
+    n_inputs = 1000
+    task_time = 1. / n_inputs
+
+    # Use default backend
+    start_time = time.time()
+    Parallel(n_jobs=n_jobs)(
+        delayed(time.sleep)(task_time) for i in range(n_inputs))
+    ref_time = time.time() - start_time
```

Comment:
You redefine ref_time here and start_time a few lines below. In my mind they were to compare that:
```py
Paralel(...)(...)
```
was taking roughly the same time as
```py
p = Parallel(...)
p(...)
```

That would be nice to do that.

Reply:
Indeed, my bad, that was my initial intention. I just forgot to remove the first `Parallel()` part...